packages-generation/manifest-validator.ps1

```diff
@@ -38,19 +38,16 @@ function Test-DownloadUrl {
     }
 }
 
-Write-Host "Downloading manifest json from '$ManifestUrl'..."
-try {
-    $manifestResponse = Invoke-WebRequest -Method Get -Uri $ManifestUrl -Headers $webRequestHeaders -MaximumRetryCount 5 -RetryIntervalSec 10
```

Comment:
It looks like we can get rid of the `webRequestHeaders` variable. We also can move the `authorizationHeaderValue` variable to the `Test-DownloadUrl` function.